koans/src/java8/AboutMultipleInheritance.java

```diff
@@ -0,0 +1,37 @@
+package java8;
+
+import com.sandwich.koan.Koan;
+
+import static com.sandwich.util.Assert.assertEquals;
+import static com.sandwich.koan.constant.KoanConstants.__;
+
+interface Human{
+    default String sound(){
+        return "hello";
+    }
+}
+
+interface Bull{
+    default String sound(){
+        return "moo";
+    }
+}
+
+class Minotaur implements Human, Bull{
+    //both interfaces implement same default method
+    //has to be overridden
```

Comment:
I love this example!  Should we ask the student to comment out the overridden `sound` method to see what happens?